src/react/components/pages/homepage/homePage.tsx

```diff
@@ -103,9 +103,11 @@ export default class HomePage extends React.Component<IHomepageProps> {
         );
     }
 
-    private createNewProject = () => {
+    private createNewProject = (e: SyntheticEvent) => {
         this.props.actions.closeProject();
         this.props.history.push("/projects/create");
+
+        e.preventDefault();
```

Comment:
Part 4 - This was the main culprit of the issue navigating to the new project screen.  This button/link is an HTML `<a/>` tag that had a `#` in the `href` attribute.  We don't actually want the browser to navigate using this value so we need to call `e.preventDefault()` to prevent the navigation from occurring and just use our `history.push(...)` call.